common/customtypes/exhibitions/index.json

```diff
@@ -183,7 +183,7 @@
               "config": {
                 "label": "Article",
```

Comment:
Should/could the label be modified?

Reply:
I've changed to 'Story or Story series' as webcomics are a type of story now - does that sound right to you?

Reply:
Yeah I think that works! It's mostly for editors so it's good to mention series too!